packages/renderer/src/lib/container/ContainerColumnName.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+import { router } from 'tinro';
+
+import type { ContainerGroupInfoUI, ContainerInfoUI } from './ContainerInfoUI';
+import { ContainerGroupInfoTypeUI } from './ContainerInfoUI';
+
+export let object: ContainerInfoUI | ContainerGroupInfoUI;
+
+function displayContainersCount(containerGroup: ContainerGroupInfoUI) {
+  let result = containerGroup.allContainersCount + ' container' + (containerGroup.allContainersCount > 1 ? 's' : '');
+  if (containerGroup.containers.length !== containerGroup.allContainersCount) {
+    result += ` (${containerGroup.allContainersCount - containerGroup.containers.length} filtered)`;
+  }
+  return result;
+}
+
+function openGroupDetails(containerGroup: ContainerGroupInfoUI) {
```

Comment:
Why is the linter not complaining about the return type not provided ?

Reply:
Good catch, I didn't notice while doing copy/paste. Maybe we don't check .svelte in renderer? I fixed all that I could find in the new commit.